Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [workspace]
-members = [
-    "meilisearch-http",
-    "meilisearch-error",
-]
+members = ["meilisearch-http", "meilisearch-error"]
```

Comment:
This change is unrelated, I'm using an automatic toml formatter and this changed was made. Happy to revert if this proves to be a problem.